### ISSUE TITLE
Acoustic Piano Soundfont Addition

### ISSUE THREAD

**Issue by Mark-Agent003** [LMMS/samples#32]
I have a completely free acoustic piano soundfont that I found here: http://zenvoid.org/audio/
I have made some changes to it, such as the name, date, etc.
The license can be found here: http://freepats.zenvoid.org/sf2/acoustic_piano_imis_1.txt
The license makes it very clear that anybody can edit or redistribute however they like. 
If I've missed something, please tell me.

**Comment by Hussam-Eddin-Alhomsi** [comment]
The soundfont is licensed under the [WTFPL](https://en.wikipedia.org/wiki/WTFPL) license which is a [public domain equivalent license](https://en.wikipedia.org/wiki/Public_domain_equivalent_license) just like the [CC0](https://en.wikipedia.org/wiki/Creative_Commons_license#Zero_.2F_public_domain) license but in the README file of the repository you can see that the first rule says that the samples must be licensed under the CC0 license (I didn't make that rule but I agree with it).
The CC0 license is better, more famous and more formal than the  WTFPL license so IMO, we should stick to it.
I _think_ that you can re-distribute the file under the C00 license because the WTFPL isn't a [copyleft](https://en.wikipedia.org/wiki/Copyleft) license.

**Comment by tresf** [comment]
@Mark-Agent003 thanks for sharing.

CC0, WTFPL and Public Domain should all be compatible.  Unfortunately, 132MB exceeds the maximum size GitHub would allow and would also make our installer extraordinary in size.  If we start a mega-pack, we should leverage GitHub's [large file storage](https://git-lfs.github.com/) instead.

Note, the author has a copyright notice in the license however CC0, WTFPL and Public Domain waves these rights.

**Comment by Hussam-Eddin-Alhomsi** [comment]
@tresf
Yes they're compatible, but don't you think that we should stick to CC0 since it's more standardized and the WTFPL is informal compared to the other public domain equivalent licenses?

**Comment by Mark-Agent003** [comment]
@tresf, thank you, I had noticed the file size issue, but it went over my head. I could make a pack that included more than just a piano such as violin, trumpet, flute, trombone, and a few other acoustic instruments as a whole bank of acoustics. Is this what you mean by a mega-pack?

**Comment by Mark-Agent003** [comment]
If it comes to it, would we be able to have a page on the LMMS website dedicated to downloading additional 'official LMMS' soundfonts, sample packs, and other large files that may not be convenient in the installer?

**Comment by tresf** [comment]
@Mark-Agent003 we would stick to our CC0 standard and re-license like you've proposed.  Public Domain and WTFPL waive all rights, we can relicense however we want including removing the confusing "copyright" attribution if we wish.

**Comment by tresf** [comment]
> If it comes to it, would we be able to have a page on the LMMS website dedicated to downloading additional 'official LMMS' soundfonts, sample packs, and other large files that may not be convenient in the installer?

We should offer an official megapack.  Let's make it official if you're ok with getting started with large file storage.  There are giga patch banks which will really help make that instrument useful as well.  We can package and host it in `samples/releases` when ready.

**Comment by tresf** [comment]
> violin, trumpet, flute, trombone, and a few other acoustic instruments as a whole bank of acoustics. Is this what you mean by a mega-pack?

Yes.

**Comment by Mark-Agent003** [comment]
![image](https://user-images.githubusercontent.com/25230510/27102120-bf16d0fe-5049-11e7-9d38-66c7a29b96f6.png)
This is what I have so far. I only included *acoustic* soundfonts. Right now it's called "LMMSMegaPack.sf2" and the file size is 545 MB.

**Comment by Mark-Agent003** [comment]
I got everything but the piano from the VSCO community edition, and (as you can see here:http://vis.versilstudios.net/vsco-2.html) it is CC0/public domain

**Comment by tresf** [comment]
@Mark-Agent003 I think you misunderstood.  A huge SF2 can really slow down loading time.  I meant a pack (e.g. zip file) of banks that could be extracted.

**Comment by Mark-Agent003** [comment]
Ok, that is definately a better way to do it. So each individual instrument has their own file, and the whole zip will be called something like "LMMS MegaPack", right?

**Comment by tresf** [comment]
Correct.

**Comment by Mark-Agent003** [comment]
Alright, I've got a .zip ready.

**Comment by Mark-Agent003** [comment]
Where should I drop it?

**Comment by tresf** [comment]
Please fork the samples repo if you haven't already and play around with the releases area.

I think the banks should live in LFS like previously mentioned and we can somehow incorporate into our samples repo.

**Comment by Mark-Agent003** [comment]
I have them on my fork: https://github.com/Mark-Agent003/samples

**Comment by tresf** [comment]
@Mark-Agent003 great.  Let's put them into a separate main folder though so we can easily distinguish them from what we will be shipping with LMMS in the future.  Please feel free to keep the exact same directory structure.
